app/graphql/mutations/create_covid_run_mutation.rb

```diff
@@ -15,10 +15,19 @@ def resolve(flowcells:)
       factory = Ont::RunFactory.new(flowcells.to_a)
 
       if factory.save
+        send_messages(run: factory.run)
```

Comment:
Should this be the other way round? Not sure just asking the question.

Reply:
What do you mean? Asking the run factory to send messages?

Reply:
As in putting send_messages after returning the response. I don't know if that is possible.

Reply:
I also do not think this is possible. Is it an issue if we send the message first? This seems to be how things are done in the saphyr and pacbio controllers

Reply:
We do that in the other pipelines too but now I have realised you can add an after hook! Not really a problem if it can't be changed just thought it might slow the render down. Not really a problem.